crates/intrinsic-test/src/types.rs

```diff
@@ -288,82 +290,79 @@ impl IntrinsicType {
         }
     }
 
-    /// Generates a comma list of values that can be used to initialize the array that
-    /// an argument for the intrinsic call is loaded from.
+    /// Generates an initialiser for an array, which can be used to initialise an argument for the
+    /// intrinsic call.
+    ///
     /// This is determistic based on the pass number.
     ///
     /// * `loads`: The number of values that need to be loaded from the argument array
     /// * e.g for argument type uint32x2, loads=2 results in a string representing 4 32-bit values
     ///
     /// Returns a string such as
-    /// * `0x1, 0x7F, 0xFF` if `language` is `Language::C`
-    /// * `0x1 as _, 0x7F as _, 0xFF as _` if `language` is `Language::Rust`
+    /// * `[0x1, 0x7F, 0xFF]` if `language` is `Language::C`
```

Comment:
Should this be `{}` instead?

Reply:
Fixed in (force-pushed) https://github.com/rust-lang/stdarch/commit/326fe6f04c8393cbdfe66cdbf8a673dde14600b3.